scripts/build-android.js

```diff
@@ -68,7 +68,7 @@ if (options.clean) {
 }
 
 if (!fs.existsSync(buildPath)) {
-  fs.mkdirSync(buildPath);
+  fs.ensureDirSync(buildPath, { recursive: true });
```

Comment:
Since `ensureDirSync` checks the `buildPath` before creation, the `if` statement above, could probably be removed.